ftp/src/main/scala/akka/stream/alpakka/ftp/impl/SftpOperations.scala

```diff
@@ -87,7 +89,7 @@ private[ftp] trait SftpOperations { _: FtpLike[JSch, SftpSettings] =>
     handler.get(name)
   }
 
-  def storeFileStream(name: String, handler: Handler): Try[OutputStream] = Try {
-    handler.put(name)
+  def storeFileStream(name: String, handler: Handler, append: Boolean): Try[OutputStream] = Try {
+    if (append) handler.put(name, APPEND) else handler.put(name, OVERWRITE)
```

Comment:
`handler.put(name, if (append) APPEND else OVERWRITE)` would read more clearly to me, minor detail though